include/nbl/builtin/shader/loader/mtl/fragment_impl.glsl

```diff
@@ -74,7 +74,7 @@ vec4 nbl_sample_Ns(in vec2 uv, in mat2 dUV) { return texture(map_Ns, uv); }
 vec4 nbl_sample_d(in vec2 uv, in mat2 dUV) { return texture(map_d, uv); }
 #endif
 #ifndef _NBL_bump_SAMPLE_FUNCTION_DEFINED_
-vec4 nbl_sample_bump(in vec2 uv, in mat2 dUV) { return texture(map_bump, uv); }
+vec4 nbl_sample_bump(in vec2 uv, in mat2 dUV) { return texture(map_bump, uv) * 2.f - vec4(1.f); }
```

Comment:
many more shaders need this fix after your change in `CDerivatieMapCreator`

Also why is this function working on and returning a `vec4` !? Should be a `vec2`